tests:
  - healthStatus:
      status: Degraded
      message: "Unable to fetch data from Datadog: Processing data from API failed, reason: no serie was found for this query in API Response, check Cluster Agent logs for QueryIndex errors, query was: max:foo.bar.metric"
    inputPath: testdata/degraded_error.yaml
  - healthStatus:
      status: Healthy
      message: "DatadogMetric is healthy"
    inputPath: testdata/healthy.yaml 